spec/services/food_data_service_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe FoodDataService do
+  it 'exists' do
+    fds = FoodDataService.new
+
+    expect(fds).to be_a(FoodDataService)
+  end
+
+  describe 'class methods' do
+    context '::search_foods' do 
+      it 'returns list of foods that include the query keyword' do
+        VCR.use_cassette("sweet_potato_search") do
+          response = FoodDataService.search_foods("sweet potato")
+
+          expect(response).to have_key(:totalHits)
+          expect(response).to have_key(:currentPage)
+          expect(response).to have_key(:pageList)
+          expect(response[:totalHits]).to eq(45635)
+          expect(response[:foods].size).to eq(50)
+          expect(response[:foods].first[:description]).to eq("SWEET POTATO")
```

Comment:
Generally, in the service spec we only need to test the structure of our JSON object rather than the individual values. Values are likely to change, but the api design likely will not change. If it does, that's something we want the test to catch.